src/utils/files/glob-source.js

```diff
@@ -0,0 +1,112 @@
+'use strict'
+
+const fs = require('fs')
+const Path = require('path')
+const isString = require('lodash/isString')
+const pull = require('pull-stream')
+const glob = require('glob')
```

Comment:
can't we just use https://www.npmjs.com/package/fast-glob ? it's faster and supports streams so we can go directly to pull-stream

Reply:
I tried it, and it's not significantly faster than `glob`, in fact in most tests it's slower (left is `glob`, right is `fast-glob`).

<img width="1440" alt="screenshot 2018-12-13 at 15 49 43" src="https://user-images.githubusercontent.com/152863/49949998-d68c7180-feee-11e8-87e7-f197212c0370.png">

Reply:
even using streams instead of glob events ? seems weird but if its faster like this im good with it

Reply:
I know weird, maybe it is the stream to pull stream conversion? Anyway even weirder is that the benchmarks for `fast-glob` do not run against `glob` 😱 They benchmark `globby`, `bash-glob`, `tiny-glob` and `glob-stream`...but not regular `glob` 🙄.